socorro/external/es/crashstorage.py

```diff
@@ -293,7 +293,9 @@ def __init__(
 
         self.client = self.build_client(url=url, timeout=timeout)
 
-        self.metrics = markus.get_metrics(metrics_prefix)
+        # Create a MetricsInterface that includes the base prefix plus the prefix passed
+        # into __init__
+        self.metrics = markus.get_metrics(build_prefix(METRICS.prefix, metrics_prefix))
```

Comment:
The `METRICS.prefix` will be either `""` or `"socorro"` depending on the cloud environment. `build_prefix` will append the crash storage metrics prefix (i.e. `"processor.es"` here) to that and that results in the prefix it uses.

Reply:
I wrote up a bug in Markus to make this a little easier. It'd be nice to hone a `MetricsInterface` iteratively. Then we wouldn't need this wonky looking thing.

https://github.com/willkg/markus/issues/142

Reply:
this causes the host tag to be lost for socorro.processor.es.* metrics in GCP. For example i got these two log lines when testing, the first with host tag working for gcs storage, the second with host tag missing for es:
```
processor-1      | 2024-05-24 15:20:13,012 INFO - processor - markus - Thread-2 - METRICS|2024-05-24 15:20:13|timing|socorro.processor.storage.save_processed_crash|12.225023994687945|#host:160ce9ffa6cc
processor-1      | 2024-05-24 15:20:13,039 INFO - processor - markus - Thread-1 - METRICS|2024-05-24 15:20:13|histogram|socorro.processor.es.crash_document_size|4454|
```

Reply:
note, this only breaks the host tag for metrics emitted in this file, metrics emitted in the base class still get those host tag, for example the first metric here has no host tag, but the second does have it:
```
processor-1      | 2024-05-24 15:20:13,148 INFO - processor - markus - Thread-2 - METRICS|2024-05-24 15:20:13|histogram|socorro.processor.es.index|95.75057029724121|#outcome:successful
processor-1      | 2024-05-24 15:20:13,148 INFO - processor - markus - Thread-2 - METRICS|2024-05-24 15:20:13|timing|socorro.processor.es.save_processed_crash|136.01990399183705|#host:160ce9ffa6cc
```

Reply:
Oh, whoops. It's because I'm not bringing over the filters, too. Good catch!